plugins/modules/data_inputs_monitors.py

```diff
@@ -0,0 +1,290 @@
+        type: str
+      host_segment:
+        description:
+          - Use the specified slash-separate segment of the filepath as the host field value.
+        type: int
+      ignore_older_than:
+        description:
+          - Specify a time value. If the modification time of a file being monitored
+            falls outside of this rolling time window, the file is no longer being monitored.
+          - This parameter is not returned back by Splunk while obtaining object information.
+            It is therefore left out while performing idempotency checks
+        type: str
+      index:
+        description:
+          - Which index events from this input should be stored in. Defaults to default.
+        type: str
+        default: "default"
+      recursive:
+        description:
+          - Setting this to False prevents monitoring of any subdirectories encountered within this data input.
+        type: bool
+      rename_source:
+        description:
+          - The value to populate in the source field for events from this data input.
+            The same source should not be used for multiple data inputs.
+          - This parameter is not returned back by Splunk while obtaining object information.
+            It is therefore left out while performing idempotency checks
+        type: str
+      sourcetype:
+        description:
+          - The value to populate in the sourcetype field for incoming events.
+        type: str
+      time_before_close:
+        description:
+          - When Splunk software reaches the end of a file that is being read, the
+            file is kept open for a minimum of the number of seconds specified in
+            this value. After this period has elapsed, the file is checked again for
+            more data.
+          - This parameter is not returned back by Splunk while obtaining object information.
+            It is therefore left out while performing idempotency checks
+        type: int
+      whitelist:
+        description:
+          - Specify a regular expression for a file path. Only file paths that match this regular expression are indexed.
+        type: str
+
+  running_config:
+    description:
+    - The module, by default, will connect to the remote device and retrieve the current
+      running-config to use as a base for comparing against the contents of source.
+      There are times when it is not desirable to have the task get the current running-config
+      for every task in a playbook.  The I(running_config) argument allows the implementer
+      to pass in the configuration to use as the base config for comparison. This
+      value of this option should be the output received from device by executing
+      command.
+    type: str
+  state:
+    description:
+    - The state the configuration should be left in
+    type: str
+    choices:
+    - merged
+    - replaced
+    - deleted
+    - gathered
+    default: merged
+
+author: Pranav Bhatt (@pranav-bhatt)
```

Comment:
plz update the author as: ` Ansible Security Automation Team (@pranav-bhatt) <https://github.com/ansible-security>`